src/services/contracts/baseDAO/registryDAO/index.ts

```diff
@@ -1,19 +1,23 @@
-import {Expr, Parser, unpackDataBytes} from "@taquito/michel-codec";
-import {TezosToolkit} from "@taquito/taquito";
-import {Schema} from "@taquito/michelson-encoder";
-import {BaseDAO, BaseDAOData, getContract} from "..";
-import {RegistryProposeArgs} from "./types";
-import {bytes2Char, char2Bytes} from "@taquito/tzip16";
+import { Expr, Parser, unpackDataBytes } from "@taquito/michel-codec";
+import { TezosToolkit } from "@taquito/taquito";
+import { Schema } from "@taquito/michelson-encoder";
+import { BaseDAO, BaseDAOData, getContract } from "..";
+import { RegistryProposeArgs } from "./types";
+import { bytes2Char, char2Bytes } from "@taquito/tzip16";
 import proposeCode from "./michelson/propose";
-import {RegistryExtraDTO} from "services/indexer/types";
-import {mapTransfersArgs} from "services/indexer/dao/mappers/proposal";
-import {BigNumber} from "bignumber.js";
-import {formatUnits} from "../../utils";
+import { RegistryExtraDTO } from "services/indexer/types";
+import { mapTransfersArgs } from "services/indexer/dao/mappers/proposal";
+import { BigNumber } from "bignumber.js";
+import { formatUnits } from "../../utils";
 
 const parser = new Parser();
 
 interface RegistryDAOData extends BaseDAOData {
   extra: RegistryExtraDTO;
+  dao_type: {
+    id: number;
+    name: string;
+  };
```

Comment:
If this is specifically a RegistryDAODAta interface, why is there a dao_type field?